pkg/whitesource/whitesource.go

```diff
@@ -317,50 +275,6 @@ func (s *System) GetProjectVulnerabilityReport(projectToken string, format strin
 	return respBody, nil
 }
 
-// GetOrganizationProductVitals
-func (s *System) GetOrganizationProductVitals() ([]Product, error) {
-	wsResponse := struct {
-		ProductVitals []Product `json:"productVitals"`
-	}{
-		ProductVitals: []Product{},
-	}
-
-	req := Request{
-		RequestType: "getOrganizationProductVitals",
-	}
-
-	respBody, err := s.sendRequest(req)
-	if err != nil {
-		return nil, errors.Wrap(err, "WhiteSource request failed")
-	}
-
-	err = json.Unmarshal(respBody, &wsResponse)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to parse WhiteSource response")
-	}
-
-	return wsResponse.ProductVitals, nil
-}
-
-// GetProductByName
-func (s *System) GetProductByName(productName string) (*Product, error) {
```

Comment:
why removing this? we should keep it.

Reply:
It only looks like that. The function was duplicated. I renamed the first instance, which was named GetMetaInfoForProduct, see above.

Reply:
ok